exporter/trace/cloudtrace.go

```diff
@@ -229,6 +253,44 @@ type Exporter struct {
 	traceExporter *traceExporter
 }
 
+// NewExportPipeline sets up a complete export pipeline with the recommended setup 
+// for trace provider. Returns provider, flush function, and errors.
+func NewExportPipeline(opts ...Option) (apitrace.Provider, func(), error) {
```

Comment:
This doesn't include an option to set the resource of the trace provider as per `WithResource`: https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/trace/provider.go#L211

In general, I think you followed the Jaeger example, but it might be better to look at how a similar function is implemented in the operations metric exporter, see: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/master/exporter/metric/metric.go#L115

It allows you to pass in sdk `push.Options` and applies them directly. You could do something similar here, but for traces, meaning you provide a way for users to specify all current and future sdktrace options (you wouldn't need to add SDKConfig, Resource, etc).

i.e. something like:
```go
func NewExportPipeline(opts []Option, topts ...sdktrace.Option) (*apitrace.Provider, func(), error) {
	...

	tp, err := sdktrace.NewProvider(
		append([]sdktrace.Option{
			sdktrace.WithSyncer(&Exporter{traceExporter: te},
		}, topts...)...,
	)
```

It would also be nice to separate the registration of the trace provider in an `InstallNewPipeline` function similar to what's done in the metric exporter.